### PR TITLE
UI adjustments

### PR DESCRIPTION
**Pull request by KevinHQi**

### CODE REVIEW

**Review by PENGYUXIONG — APPROVED** [review]
looks good

**Review by AnkushSharma2698 — APPROVED** [review]
Looks good

File: frontend/src/containers/layout.js [hunk 1]
```diff
@@ -52,47 +57,132 @@ class CustomLayout extends React.Component {
                 this.inboxMsg = this.props.inbox.items.items;
             }
         }
+
+        if (this.props.isAuthenticated){
+            this.props.getMyProfile();
```
Comment (PENGYUXIONG) [inline frontend/src/containers/layout.js:66]: Why adding this line?

Comment (PENGYUXIONG) [reply frontend/src/containers/layout.js:66]: This line will send too many requests, need more conditions before dispatch actions.

remove it if not need

**Review by JialinX — APPROVED** [review]

**Pull request merged**